Fix FileColumn documentation and changelog

graup opened the pull request:
Just noticed the text was not quite right. Also added it to the changelog and fixed another typo there.

jieter commented on CHANGELOG.md:
```diff
@@ -1,13 +1,13 @@
 
 ## Unreleased
-
+- Accept `text` parameter in `FileColumn` like in `BaseLinkColumn` (#343)
```
I usally use something like this:
```
 - Add `text` attribute to `FileColumn`, analogous to `LinkColumn` ([#343](https://github.com/bradleyayers/django-tables2/pull/343) by [@graup](https://github.com/graup))
```

graup commented:
Fixed. I changed the wording to be like some other release notes.

By the way, thanks for showing the project some love recently. This is one of my favorite Django apps and I am happy to see it be actively maintained.

jieter commented:
Thanks, merged.

Appreciate your words, thanks!
